lib/options-handler.ts

```diff
@@ -245,7 +357,9 @@ export class ClientOptionsHandler {
         for (const langGroup of Object.values(libraries)) {
             for (const libGroup of Object.values(langGroup)) {
                 const versions = Object.values(libGroup.versions);
-                versions.sort((a, b) => semverParser.compare(asSafeVer(a.semver), asSafeVer(b.semver), true));
+                versions.sort((a, b) =>
+                    semverParser.compare(asSafeVer((a as any).semver), asSafeVer((b as any).semver), true),
```

Comment:
`a` and `b` here are `VersionInfo`'s and nowhere above is a `semver` property set. Pretty sure this is a bug / no-op but it might be safer to leave it for now.

Reply:
agreed! thanks for taking the "note, but move on" aprroach here.

Reply:
This might be my bad, but this definetely used to work, so yeah definetely something weird going on here